### ISSUE TITLE
Link text/ anchor text would be great to have

### ISSUE THREAD

**Issue by mlapeter** [jhollingworth/bootstrap-wysihtml5#42]
I saw you mentioned this in a previous issue, but being able to choose the link text would be great, as many users like to select their own link text, especially as it helps with seo. Thanks for considering!

**Comment by bencrouse** [comment]
+1 

I saw the other issue also, but on the demo, selecting text and clicking the button to insert the link doesn't retain the selection of the text.

**Comment by jhollingworth** [comment]
The functionality isn't currently exposed via the wysihtml5 API. I'm happy to implement it but just checking with them first to see if they are happy to include it (https://github.com/xing/wysihtml5/issues/42)

**Comment by bencrouse** [comment]
Yeah, I browsed through the wysihtml5 API and couldn't figure out how to do it.
But, I wonder how it works in their advanced demo then. http://tifftiff.de/wysihtml5/examples/advanced.html

**Comment by jhollingworth** [comment]
My pull request has been accepted (https://github.com/xing/wysihtml5/pull/45). I will add the needed functionality asap

**Comment by zzack** [comment]
Is this issue to add a text box on the insert link button or to preserve the selection before the button was clicked and apply the link to it?

**Comment by mlapeter** [comment]
I would say both. Since it already automatically turns www.example.com in the text into a link, the only reason I could see a user selecting text and then creating a link would be if they wanted to keep the original text. 

Alternatively If they haven't selected text and are clicking the button, it would also be nice to let them choose a title, otherwise some users may never realize it's even an option.  Wordpress for example has two fields in the popup, URL and Title. And thanks James for the quick response!

**Comment by ManuelThurner** [comment]
has this already been implemented in the project? if not, can anyone tell me how to add it quickly to my project since it is already part of the original wysihtml5 project? thanks
